_data/config.yml

```diff
@@ -1,3 +1,3 @@
-grpc_release_branch: v1.0.0
```

Comment:
I am reluctant to changing it in this tiny commit as it is referenced in a dozen places and might need more thought and knowledge of project history.